### ISSUE TITLE
Gradle plugin

### ISSUE THREAD

**Issue by stboiss** [CycloneDX/cyclonedx-maven-plugin#3]
is there a plugin "gradle" planned?

**Comment by stevespringett** [comment]
Yes, keep an eye out for https://github.com/CycloneDX/cyclonedx-gradle-plugin

I've had one partially working for a few weeks now, but I'll likely be finishing it up the first week of June.

**Comment by stboissdev** [comment]
@stevespringett thank's
